tests/test_models/test_nn/deepar/test_deepar_new.py

```diff
@@ -0,0 +1,97 @@
+from unittest.mock import MagicMock
+
+import numpy as np
+import pytest
+from pytorch_lightning import seed_everything
+
+from etna.metrics import MAE
+from etna.models.nn import DeepARModelNew
+from etna.models.nn.deepar_new.deepar import DeepARNetNew
+from etna.transforms import StandardScalerTransform
+from tests.test_models.utils import assert_model_equals_loaded_original
+from tests.test_models.utils import assert_sampling_is_valid
+
+
+@pytest.mark.parametrize(
+    "horizon",
+    [
+        8,
+        13,
+        15,
+    ],
+)
+def test_deepar_model_run_weekly_overfit(ts_dataset_weekly_function_with_horizon, horizon):
+    """
+    Given: I have dataframe with 2 segments with weekly seasonality with known future
+    Then: I get {horizon} periods per dataset as a forecast and they "the same" as past
+    """
+    seed_everything(0, workers=True)
+    ts_train, ts_test = ts_dataset_weekly_function_with_horizon(horizon)
+    std = StandardScalerTransform(in_column="target")
+    ts_train.fit_transform([std])
+    encoder_length = 14
+    decoder_length = 14
+    model = DeepARModelNew(
+        input_size=1,
+        encoder_length=encoder_length,
+        decoder_length=decoder_length,
+        scale=False,
+        trainer_params=dict(max_epochs=100),
+    )
+    future = ts_train.make_future(horizon, transforms=[std], tail_steps=encoder_length)
+    model.fit(ts_train)
+    future = model.forecast(future, prediction_size=horizon)
+    future.inverse_transform([std])
+
+    mae = MAE("macro")
+    assert mae(ts_test, future) < 0.06
+
+
+@pytest.mark.parametrize("scale, mean_1, mean_2", [(False, 0, 0), (True, 4.439109105024682, 5.516483350680801)])
```

Comment:
Better to use some simple fixture where the weight are clear, now "4.439109105024682" is just a magic number